src/js/messages/i18n/de_DE.js

```diff
@@ -0,0 +1,48 @@
+(function($) {
+
+	// german - germany
+
+	var Alpaca = $.alpaca;
+
+	Alpaca.registerView ({
+		"id": "base",
+		"messages": {
+            "de_DE": {
+                required: "Eingabe erforderlich",
+                invalid: "Eingabe ungültig",
+                months: ["Jänner", "Februar", "März", "April", "Mai", "Juni", "Juli", "August", "September", "Oktober", "November", "Dezember"],
```

Comment:
Januar is the proper German (i.e de_DE).  Jänner is used regionally. I'm not sure if Jänner is the official usage in Austria (i.e. at_DE) but I know for certain it's not in Germany.